stdlib/public/core/OutputStream.swift

```diff
@@ -638,3 +638,5 @@ internal struct _TeeStream<L: TextOutputStream, R: TextOutputStream>
   internal mutating func _lock() { left._lock(); right._lock() }
   internal mutating func _unlock() { right._unlock(); left._unlock() }
 }
+
+internal let noReflectionMetadataErrorMessage = "(value cannot be printed without reflection)"
```

Comment:
I will revert this. Can you explain/link to why the stdlib avoids globals? I would like to understand if the reason is exclusive to the stdlib or something that is applicable outside.

Reply:
Globals have a place in the language, but I think we would reserve them for things like global caches if we had any of those (like the ones in the runtime). This is simple enough that I don't think it's worth having one.

Reply:
Ah, interesting. My guess was that it would be a perf issue related to initializing the let in a thread safe way. 

My goal with making this change was to prevent code duplication, but it's definitely not a particularly serious issue if this string gets copy pasted around a few times.